source/application/AMDatamanAppController.cpp

```diff
@@ -873,6 +873,8 @@ bool AMDatamanAppController::startupPopulateNewDatabase()
 	bioXASImaging.storeToDb(db);
 	AMFacility sxrmb("SXRMB", "CLS SXRMB Beamline", ":/clsIcon.png");
 	sxrmb.storeToDb(db);
+	AMFacility pgm("PGM", "CLS PGM Bealine", ":/clsIcon.png");
+	pgm.storeToDb(db);
```

Comment:
@dretrex I know this is closed. Since the #483 is still under development, this changes here will have the same issue as what we had for the other beamline since we didn't define the facility id for PGM .